tests/unit/src/kakarot/instructions/test_environmental_information.cairo

```diff
@@ -140,3 +140,39 @@ func test__exec_extcodecopy__should_handle_address_with_no_code{
 
     return ();
 }
+
+@view
+func test__returndatacopy{
+    syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr, bitwise_ptr: BitwiseBuiltin*
+}() {
+    // Given
+    alloc_locals;
+
+    let (bytecode) = alloc();
+    let return_data: felt* = alloc();
+    let return_data_len: felt = 32;
+    // filling at return_data + 1 because first first felt is return_data offset
+    TestHelpers._fill_bytecode_with_values(return_data + 1, 32, 0xFF);
+    let child_ctx: model.ExecutionContext* = TestHelpers.init_context_with_return_data(
+        0, bytecode, return_data_len, return_data
+    );
+
+    let stack: model.Stack* = Stack.init();
+    let stack: model.Stack* = Stack.push(stack, Uint256(32, 0));
```

Comment:
Can you add a comment with what we are adding like defined in the playground? (size, offset, etc)